packages/unlock-js/src/index.ts

```diff
@@ -1,8 +1,6 @@
 export { default as Web3Service } from './web3Service'
 export { default as WalletService } from './walletService'
 export { getCurrentProvider, getWeb3Provider } from './providers'
-export const latestUnlock = 'v11' // TODO Are we using this?
-export const latestPublicLock = 'v11' // TODO Are we using this? (it is not even up to date)
```

Comment:
we can safely get rid of this. The code itself doesn't contain references to it + we shouldnt keep outdated variables